web/components/HOC/PageAccess.tsx

```diff
@@ -5,18 +5,27 @@ import React from "react";
 import { Center, Spinner } from "@chakra-ui/react";
 import { useRouter } from "next/router";
 import PageNotFoundError from "../404";
+import Head from "next/head";
 
 const unrestricted = new Set([Role.Volunteer, Role.ContentCreator, Role.Admin]);
 const restricted = new Set([Role.Admin]);
 
 const pageAccess: Record<Pages, Set<Role>> = {
   [Pages.ONBOARDING]: unrestricted,
   [Pages.ACCESS_MANAGEMENT]: restricted,
-  [Pages.PROFILE]: unrestricted,
+  [Pages.PROFILE]: new Set(),
```

Comment:
Curious about the change here - could you explain?

Reply:
Oops, was testing how it worked and forgot to change back